pkg/apps/watch-appstream.py

```diff
@@ -356,8 +356,12 @@ def installed_callback(path):
     def available_callback(path):
         process_file(path, lambda path, xml: db.notice_available(path, xml))
 
-    watcher.watch_directory('/usr/share/metainfo', installed_callback)
+    # https://www.freedesktop.org/software/appstream/docs/chap-CatalogData.html
     watcher.watch_directory('/usr/share/swcatalog/xml', available_callback)
+    watcher.watch_directory('/var/cache/swcatalog/xmls', available_callback)
+    watcher.watch_directory('/var/lib/swcatalog/xml', available_callback)
+    # legacy paths
+    watcher.watch_directory('/usr/share/metainfo', installed_callback)
```

Comment:
`/usr/share/metainfo` is not legacy, it's still the current location for metainfo of installed packages

Reply:
Thanks, moved to the right place.